src/pages/experiments/[experimentId]/data-exploration/components/heatmap/HeatmapPlot.jsx

```diff
@@ -299,7 +300,7 @@ const HeatmapPlot = (props) => {
       <PlatformError
         description={error}
         onClick={() => {
-          dispatch(loadGeneExpression(experimentId, selectedGenes, COMPONENT_TYPE));
+          dispatch(loadGeneExpression(experimentId, selectedGenes, COMPONENT_TYPE, expressionType));
```

Comment:
similarly to my comment for `ComponentActions`, you shouldn't need those changes here.